.travis/stages/verify/check-links-and-yaml-syntax/check-links

```diff
@@ -15,7 +15,7 @@ fi
 
 function checkUri() {
   local -r uri="$1"
-  curl -s --head "$uri" | head -n 1  | grep -qE  "HTTP/2 200|(200|302) (OK|Found)" \
+  curl -s --head "$uri" | head -n 1  | grep -qE "HTTP/(2 200|1.1 (200 OK|302 Found))" \
     || reportBrokenUri "$uri"
```

Comment:
I think positive confirmation would be helpful to know what we are checking and are seeing. I'd also recommend we try and keep it simple. Perhaps there is no need to even check the HTTP type, just status code and the optional last word.

WDYT of something like this?

```
  echo -n "$uri -- "  
  local -r curlResult=$(curl -s --head "$uri" | head -n 1)
  echo "$curlResult"
  echo "$curlResult" | grep -qE "200( OK)?|302( Found)?" \
    || reportBrokenUri "$uri"
```